homeassistant/components/tuya/config_flow.py

```diff
@@ -0,0 +1,108 @@
+"""Config flow for Tuya."""
+import logging
+
+from tuyaha import TuyaApi
+from tuyaha.tuyaapi import TuyaAPIException, TuyaNetException, TuyaServerException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_PLATFORM, CONF_USERNAME
+
+# pylint:disable=unused-import
+from .const import CONF_COUNTRYCODE, DOMAIN, TUYA_PLATFORMS
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA_USER = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_COUNTRYCODE): vol.Coerce(int),
```

Comment:
If the only options are `1 for USA or 86 for China` we should use `vol.In` here and limit their choices to ones that are valid.

Reply:
No, this are only example (equal to the manual page). Code can be any valid country code.